library/core/src/any.rs

```diff
@@ -884,6 +884,31 @@ impl<'a> Demand<'a> {
     ///
     /// # Examples
     ///
+    /// Provides an `u8`.
+    ///
+    /// ```rust
+    /// # #![feature(provide_any)]
```

Comment:
Do we have a habit of hiding features we enable for examples? I would prefer that this is visible but if this is following existing patterns lmk

edit: I see the example below where it was already hidden. I did a quick search and it looks like we definitely have a history of doing both but not hiding the feature is significantly more common. I want to fix this but I think it's probably better to deal with it as a follow up issue than as part of this one so do not worry about it.

Reply:
I'd be happy to un-hide it for my newly-added functions if you think it's valuable!